docs/rules/sort-imports.md

```diff
@@ -36,6 +36,7 @@ This rule accepts an object with its properties as
 * `ignoreCase` (default: `false`)
 * `ignoreDeclarationSort` (default: `false`)
 * `ignoreMemberSort` (default: `false`)
+* `ignoreBlankLines` (default: `true`)
```

Comment:
this might be a breaking change, as we follow [semantic-versioning-policy](https://github.com/eslint/eslint#semantic-versioning-policy)

Reply:
I think it wouldn't be breaking, given the description in #12951, but this option's name is definitely confusing and we should change it.

Reply:
it is a breaking change, as users have to update their config to keep the current behaviour.
am I misunderstanding something?

Reply:
This will be more clear with `allowSeparatedGroups` name.

Per the original proposal, `ignoreBlankLines: true` was supposed to keep the current behavior, so it wouldn't be necessary to update configs.

I believe the logic for `ignoreBlankLines` was: blank lines should normally reset sorting, but by default this rule ignores them.

Reply:
I see, thanks! ❤️